lib/roo_on_rails/default.env

```diff
@@ -9,3 +9,8 @@ NEW_RELIC_DEVELOPER_MODE=false
 NEW_RELIC_LICENSE_KEY=override-me
 RACK_SERVICE_TIMEOUT=10
 RACK_WAIT_TIMEOUT=30
+SIDEKIQ_ENABLED=true
+SIDEKIQ_THREADS=25
+SIDEKIQ_DB_POOL_SIZE=26
```

Comment:
This is the recommendation (thread count + 1) but it means we definitely need pg_bouncer set up by roo_on_rails too!